internal/provider/cloudspace_resource.go

```diff
@@ -63,6 +63,23 @@ func (r *cloudspaceResource) Configure(ctx context.Context, req resource.Configu
 }
 
 func (r *cloudspaceResource) ModifyPlan(ctx context.Context, req resource.ModifyPlanRequest, resp *resource.ModifyPlanResponse) {
+	if req.Plan.Raw.IsNull() {
+		// resource is being destroyed
```

Comment:
Should we log this?

Reply:
This change has been reverted, so logging is no longer necessary.